test/core/integration/concrete/lockup-dynamic/token-uri/tokenURI.t.sol

```diff
@@ -35,17 +34,10 @@ contract TokenURI_LockupDynamic_Integration_Concrete_Test is LockupDynamic_Integ
         lockupDynamic.tokenURI({ tokenId: nullStreamId });
     }
 
-    modifier givenNFTExists() {
-        defaultStreamId = createDefaultStream();
```

Comment:
Since this logic has been removed from `givenNFTExists` defined in Modifiers contract, dont we need it?

Reply:
this logic is implemented:

https://github.com/sablier-labs/v2-core/blob/80de93c10a7e10a0529354c11ef8da8f3f5eeebb/test/core/integration/shared/lockup-dynamic/LockupDynamic.t.sol#L54

https://github.com/sablier-labs/v2-core/blob/80de93c10a7e10a0529354c11ef8da8f3f5eeebb/test/core/integration/shared/lockup-linear/LockupLinear.t.sol#L30

https://github.com/sablier-labs/v2-core/blob/80de93c10a7e10a0529354c11ef8da8f3f5eeebb/test/core/integration/shared/lockup-tranched/LockupTranched.t.sol#L53

Reply:
Got it.